examples/loader-python-to-parquet/src/index.md

```diff
@@ -53,7 +53,7 @@ const dams = FileAttachment("data/us-dams.parquet").parquet();
 We can display the table using `Inputs.table`.
 
 ```js echo
-Inputs.table(dams)
+Inputs.table(dams);
```

Comment:
This prettier edit will prevent the table from displaying.

Reply:
I think this was correctly updated in a subsequent commit (I turned prettier off after formatting, to leave the semicolon after FileAttachment but remove in the Inputs.table and Plot code: 

```js echo
Inputs.table(dams)
```